agent/maestro_agent/settings.py

```diff
@@ -28,6 +28,9 @@
 JMETER_BASE_IMAGE = "maestrojmeter:%s"
 JMETER_IMAGE_BASE_REPO = os.environ.get("JMETER_IMAGE_BASE_REPO", "")
 JMETER_IMAGE_BASE_VERSION = os.environ.get("JMETER_IMAGE_BASE_VERSION", "")
+JMETER_IMAGE_HEAP = os.environ.get(
+    "JMETER_IMAGE_HEAP", "-Xms2g -Xmx16g -XX:MaxMetaspaceSize=256m"
```

Comment:
Maybe we should keep the default values when HEAP is not defined? What do you think?

Default values:
`HEAP="-Xms1g -Xmx1g -XX:MaxMetaspaceSize=256m"`

Reply:
Yes, agree, makes sense for me!